test/language/statements/class/definition/fn-length-static-precedence.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-runtime-semantics-classdefinitionevaluation
+description: >
+    Function `length` attribute not inferred in presence of static `length` method
+info: |
+    ClassTail : ClassHeritage_opt { ClassBody_opt }
+
+    14. If constructor is empty, then [...]
+      b. Let F be ! CreateBuiltinFunction(steps, 0, className, « [[ConstructorKind]], [[SourceText]] », empty, constructorParent).
+    15. Else,
+      a. Let constructorInfo be ! DefineMethod of constructor with arguments proto and constructorParent.
+         [ This sets the length property on constructorInfo.[[Closure]]. ]
+      b. Let F be constructorInfo.[[Closure]].
+      [...]
+    25. For each ClassElement e of elements, do
+      a. If IsStatic of e is false, then [...]
+      b. Else,
+        i. Let field be ClassElementEvaluation of e with arguments F and false.
+           [ This overwrites the length property on F. ]
+includes: [compareArray.js]
+features: [generators]
+---*/
+
+class A {
+  static method() {
+    $ERROR('Static method should not be executed during definition');
+  }
+  static length() {
+    $ERROR('Static method should not be executed during definition');
```

Comment:
Please don't use `$ERROR` like this. `throw new Test262Error(...)` (This is clearly stated in the contributing docs https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#test-environment)

Reply:
This is primarily copied from the existing fn-name-static-precedence.js test, which explains the existence of $ERROR. I can fix it up.

Reply:
I appreciate the follow up. I just finished eliminating `$ERROR` from every test file in the project 🤣